core/modules/system/system.admin.inc

```diff
@@ -1779,6 +1779,33 @@ function system_performance_settings($form, &$form_state) {
   return system_settings_form($form);
 }
 
+/**
+ * Form builder; Configure jQuery settings.
+ *
+ * @ingroup forms
+ */
+function system_jquery_settings() {
+  $config = config('system.core');
+
+  $form['#config'] = 'system.core';
+
+  $options = array(
+    'default' => t('jQuery 3 (recommended)'),
+    'legacy' => t('jQuery 1 (legacy)'),
+  );
+  $form['jquery_version'] = array(
+    '#type' => 'radios',
+    '#title' => t('jQuery version'),
+    '#default_value' => $config->get('jquery_version'),
+    '#options' => $options,
+    '#description' => t('<a href="@url">More information about jQuery</a> can be found in the documentation.', array('@url' => url('https://docs.backdropcms.org/documentation/introducing-jquery'))),
```

Comment:
While using both individual descriptions for each option as well as a description for the entire element is possible, it is not ideal (the description text of the last item is close to the "global" description of the element). If we need this, we should change it to a separate `'#type' => 'help'` element above the radios.

Reply:
Can we also update this URL to simply https://docs.backdropcms.org/documentation/jquery for better future-proofing? I don't think we need to use the title "Introducing jQuery" for the [most used JS library](https://w3techs.com/technologies/overview/javascript_library) in the world.

Reply:
And I agree with @klonos, let's move this to a help element so we don't double-up descriptions on both individual elements and the overall radio element.